Libraries/LibWeb/HTML/FormAssociatedElement.cpp

```diff
@@ -226,6 +226,20 @@ WebIDL::ExceptionOr<void> FormAssociatedElement::set_form_action(String const& v
     return html_element.set_attribute(HTML::AttributeNames::formaction, value);
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#check-validity-steps
+WebIDL::ExceptionOr<bool> FormAssociatedElement::check_validity()
```

Comment:
It might be better to rename this method to `check_validity_steps()` to avoid the method being hidden. This would also match the name used in the spec.

I don't think there's any need for this to return `ExceptionOr` since there's no way it can error.